### PR TITLE
Fix input with reply goes down

### PR DESCRIPTION
**Pull request by Johnnycus**

### CODE REVIEW

File: viewportHax.js [hunk 1]
```diff
@@ -3,48 +3,48 @@
...
+  view.style.paddingTop = `${isUp ? 0 : canvas.offsetHeight}px`;
 
-    const navHeight = document.querySelector("nav").offsetHeight;
-    const footerHeight = document.querySelector("footer").offsetHeight;
-    const vpH = window.visualViewport.height;
-    view.style.height = `${vpH - navHeight - footerHeight}px`;
+  const navHeight = document.querySelector("nav").offsetHeight;
+  const footerHeight = document.querySelector("footer").offsetHeight;
+  const vpH = window.visualViewport.height;
+  view.style.height = `${vpH - navHeight - footerHeight - 15}px`;
```
Comment (Johnnycus) [inline viewportHax.js:17]: added -15 to add a bit of space below the message box

Comment (billythedummy) [reply viewportHax.js:17]: nice i was just thinking the same

File: minimalTest.js [hunk 2]
```diff
@@ -133,17 +133,21 @@ const messageData = {
     }
   },
   handleReply() {
+    const view = document.getElementById("view");
+    view.style.height = `${view.clientHeight - 70}px`;
```
Comment (billythedummy) [inline minimalTest.js:137]: nit: lets leave a comment about this 70px being for the reply display?

**Review by billythedummy — COMMENTED** [review]
does this break when replying to images?

**Comment by Johnnycus** [comment]
> does this break when replying to images?

nope, image thumbnail doesn't increase the space of reply box

**Review by billythedummy — APPROVED** [review]

**Comment by Johnnycus** [comment]
added a fix so we dont add/remove 70px when you select/deselect multiple times. only when reply message is not selected/selected

**Pull request merged**